modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/bnd.bnd

```diff
@@ -2,14 +2,15 @@ Bundle-Name: ${artifactId}
 Bundle-SymbolicName: ${package}
 Bundle-Version: ${version}
 Export-Package: ${package}.constants
-
 -includeresource:\
 	@com.liferay.util.taglib-[0-9\.]*.jar!/META-INF/*.tld
--privatepackage: com.liferay.util.bridges.freemarker#if (${buildType} != "gradle")
-
+#if (${buildType} != "gradle")
 #if (${liferayVersion.startsWith("7.1")} || ${liferayVersion.startsWith("7.2")})-contract: JavaPortlet,JavaServlet
```

Comment:
Can we move `-contract: ...` before `-includeresource: ...` so its sorted alphabetically everywhere?